### DISCUSSION TITLE
alpine-docker not showing docker stats

### DISCUSSION THREAD

**Discussion by hlugt** [tteck/Proxmox#1403] [Help Needed]
### A clear and concise description of the issue.

Hello,
Running privileged lxc with alpine-docker is not showing stats for running docker containers. 
Proxmox 7.4.3 with pve-kernel-6.2 
Please advice what is missing...

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

![image](https://user-images.githubusercontent.com/7116919/236809264-0186aaa0-1b91-4a62-8268-6e8e8ed9e12a.png)

### Please provide detailed steps to reproduce the issue.

Create docker lxc with alpine-docker template. Start any docker container.
lxc conf:

```shell
## Alpine-Docker LXC
#  ### https%3A//tteck.github.io/Proxmox/
#  <a href='https%3A//ko-fi.com/D1D7EP4GF'><img src='https%3A//img.shields.io/badge/%E2%98%95-Buy me a coffee-red' /></a>
arch: amd64
cores: 4
features: mount=nfs,nesting=1
hostname: docker-plex
memory: 8192
mp0: local-zfs:subvol-102-disk-1,mp=/volume1/apps,backup=1,size=64G
net0: name=eth0,bridge=vmbr0,hwaddr=6E:50:38:08:D8:7F,ip=dhcp,type=veth
onboot: 1
ostype: alpine
rootfs: local-zfs:subvol-102-disk-0,size=2G
swap: 512
lxc.cgroup2.devices.allow: a
lxc.cap.drop: 
lxc.cgroup2.devices.allow: c 188:* rwm
lxc.cgroup2.devices.allow: c 189:* rwm
lxc.cgroup2.devices.allow: c 226:0 rwm
lxc.cgroup2.devices.allow: c 226:128 rwm
lxc.cgroup2.devices.allow: c 29:0 rwm
lxc.mount.entry: /dev/serial/by-id   dev/serial/by-id   none bind,optional,create=dir
lxc.mount.entry: /dev/ttyUSB0        dev/ttyUSB0        none bind,optional,create=file
lxc.mount.entry: /dev/ttyUSB1        dev/ttyUSB1        none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM0        dev/ttyACM0        none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM1        dev/ttyACM1        none bind,optional,create=file
lxc.mount.entry: /dev/fb0            dev/fb0            none bind,optional,create=file
lxc.mount.entry: /dev/dri            dev/dri            none bind,optional,create=dir
lxc.mount.entry: /dev/dri/renderD128 dev/dri/renderD128 none bind,optional,create=file
```

**Comment by hlugt** [comment]
Ok, changed over to a debian lxc then. Was hoping to keep it as light as possible but can not figure out how to probably allow enough access to the cgroup mounts...
And in debian/ubuntu works out-of-the-box...

**Comment by turtletowerz** [comment]
Bumping this because it's still an issue a year and a half later. It seems to be a problem with how OpenRC interacts with cgroups, but I have no knowledge of either beyond [a basic blog post on how to fix a similar issue](https://wildwolf.name/how-to-run-docker-in-alpine-container-in-lxc-lxd/).

When creating a new lxc and running
```
apk add docker && rc-update add docker default && service docker start
```

Mount errors are spammed in the console, which is weird because it looks like the cgroup directories mount properly. Any help or at least a place to start would be appreciated.
